src/dataobject.cpp

```diff
@@ -247,7 +247,7 @@ void dataobject:: get_records_pred(recstack& a, int NID, int neq,
   }
   for(int h=0; h < NID; ++h) {
     lastime = Data(this->start(h),col[_COL_time_]);
-    a[h].reserve(this->end(h) - this->start(h) + 5);
+    //a[h].reserve(this->end(h) - this->start(h) + 5); // TODO: remove
```

Comment:
There is no reserve method for `deque`. I'd like to retain these for now in case we decide to revert.